azure.yaml

```diff
@@ -1,18 +1,33 @@
 - active: false
   authorities:
-    - name: app-operator
-      version: 1.0.0
-    - name: azure-operator
-      version: 2.8.0
-    - name: cert-operator
-      version: 0.1.0
-    - name: chart-operator
-      version: 0.7.0
-    - name: cluster-operator
-      provider: azure
-      version: 0.21.1-dev
-  date: 2019-10-25T10:00:00Z
-  version: 9.1.0
+  - name: app-operator
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.9.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 2.1.0-dev
+  date: 2019-12-27T12:00:00Z
+  version: 11.1.0
+- active: true
+  authorities:
+  - name: app-operator
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 2.0.0
```

Comment:
This might be wrong? Azure doesn't have Cluster API types yet.

Reply:
Fixed.